run.sh

```diff
@@ -16,22 +16,26 @@ log_cyan "Inspiration by Raphael Murray https://github.com/raphmur"
 log_cyan "Instructions by Shankar Kumarasamy https://shankarkumarasamy.blog/2024/01/28/tesla-developer-api-guide-ble-key-pair-auth-and-vehicle-commands-part-3"
 
 
+###
+###
+### TODO
+### Add validation for input, specially for docker. Addon in config allows to specify
+### What's valid/needed or not.
+###
+###
+
 ### INITIALIZE VARIABLES AND FUNCTIONS TO MAKE THIS .sh RUN ALSO STANDALONE ##########################################
 # read options in case of HA addon. Otherwise, they will be sent as environment variables
 if [ -n "${HASSIO_TOKEN:-}" ]; then
-  export TESLA_VIN1="$(bashio::config 'vin1')"
-  export TESLA_VIN2="$(bashio::config 'vin2')"
-  export TESLA_VIN3="$(bashio::config 'vin3')"
-  export PRESENCE_DETECTION="$(bashio::config 'presence_detection')"
-  export BLE_MAC1="$(bashio::config 'ble_mac1')"
-  export BLE_MAC2="$(bashio::config 'ble_mac2')"
-  export BLE_MAC3="$(bashio::config 'ble_mac3')"
+  export BLE_MAC_LIST="$(bashio::config 'ble_mac3')"
```

Comment:
HA already have an option for config settings in the config.yaml that supports a list.
```
list(val1|val2|...)
```

They use a | as a separator, we should do the same for the list.